### ISSUE TITLE
jemalloc maybe incompatible with some aarch64 CPUs

### ISSUE THREAD

**Issue by SebastianSchildt** [eclipse/kuksa.val#460]
I can't run our ARM builds on a Cortex-A57

```
[/share/Download] # ./databroker/databroker 
<jemalloc>: Unsupported system page size
<jemalloc>: Unsupported system page size
memory allocation of 5 bytes failed
Aborted
[/share/Download] # cat /proc/cpuinfo 
processor	: 0
model name	: Annapurna Labs Alpine AL324 Quad-core ARM Cortex-A57 CPU @ 1.70GHz
Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32
CPU implementer	: 0x41
CPU architecture: 8
CPU variant	: 0x1
CPU part	: 0xd07
CPU revision	: 3

processor	: 1
model name	: Annapurna Labs Alpine AL324 Quad-core ARM Cortex-A57 CPU @ 1.70GHz
Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32
CPU implementer	: 0x41
CPU architecture: 8
CPU variant	: 0x1
CPU part	: 0xd07
CPU revision	: 3

processor	: 2
model name	: Annapurna Labs Alpine AL324 Quad-core ARM Cortex-A57 CPU @ 1.70GHz
Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32
CPU implementer	: 0x41
CPU architecture: 8
CPU variant	: 0x1
CPU part	: 0xd07
CPU revision	: 3

processor	: 3
model name	: Annapurna Labs Alpine AL324 Quad-core ARM Cortex-A57 CPU @ 1.70GHz
Features	: fp asimd evtstrm aes pmull sha1 sha2 crc32
CPU implementer	: 0x41
CPU architecture: 8
CPU variant	: 0x1
CPU part	: 0xd07
CPU revision	: 3

```

**Comment by SebastianSchildt** [comment]
Found this
https://github.com/jemalloc/jemalloc/issues/467
Need to investigate, however if there is a hard dependency on the build system, maybe making jemalloc optional is better.

I think we should not start shipping builds for different pagesizes, as most users would likely not even know what there's is

**Comment by SebastianSchildt** [comment]
Fixed in #468
